tests/src/Functional/ScaffoldTest.php

```diff
@@ -101,7 +96,7 @@ public function scaffoldFixturesThatThrowTestValues() {
     return [
       [
         'drupal-drupal-missing-scaffold-file',
-        'assertDrupalDrupalSutWasScaffolded',
+        '_no_assertion_',
```

Comment:
Why did you make this `_no_assertion_`? The test was passing here before, wasn't it?

Reply:
Yes, but it doesn't actually make an assertion.

Reply:
Ah, right, this data provider should just remove its second parameter, as its unused.